BCDA-10263: v3 Site Updates (#374)
CMSgov/bcda-static-site: src/_pages/v3/partially-adjudicated-claims-data-v3.md

All EOBs contain the <a href="https://hl7.org/fhir/R4/explanationofbenefit-definitions.html#ExplanationOfBenefit.outcome" target="_blank" rel="noopener noreferrer">ExplanationOfBenefit.outcome</a> element which will indicate if the claim is still processing. If you wish to include only EOBs that have been fully processed and paid in your job requests, see our guidance on <a href="{{ '/v3/filter-claims-data-v3.html#the-typefilter-parameter' | relative_url }}">filtering claims by `ExplanationOfBenefit.outcome` using _typeFilter</a>.

#### Matching multiple EOBs to the same claim

> In v3, claims are automatically deduplicated within a source system. You will no longer need to deduplicate multiple versions of a claim within the same System-Type.

Reviewer: Nit picking: System-Type is a new concept so users haven't ever needed to deduplicate multiple versions of a claim within the same type.
```suggestion
In v3, claims are automatically deduplicated within a source system. You won't need to deduplicate multiple versions of a claim within the same System-Type.
```
